src/runtime-rs/crates/hypervisor/src/ch/inner_device.rs

```diff
@@ -43,6 +44,14 @@ impl CloudHypervisorInner {
         match device {
             DeviceType::ShareFs(sharefs) => self.handle_share_fs_device(sharefs.config).await,
             DeviceType::HybridVsock(hvsock) => self.handle_hvsock_device(&hvsock.config).await,
+            DeviceType::Block(block) => self
+                .handle_block_device(
+                    block.config.path_on_host.as_str(),
+                    block.device_id.as_str(),
+                    block.config.is_readonly,
+                    block.config.no_drop,
+                )
```

Comment:
Nit: For consistency with the other handlers, you could just pass the `BlockConfig` to the handle_block_device()` method.

Reply:
Updated